supabase/migrations/20230411175953_langchain-hybrid_search.sql

```diff
@@ -133,6 +128,12 @@ import { OpenAIEmbeddings } from "langchain/embeddings/openai";
 import { createClient } from "@supabase/supabase-js";
 import { SupabaseHybridSearch } from "langchain/retrievers/supabase";
 
+const privateKey = process.env.SUPABASE_PRIVATE_KEY;
+if (!privateKey) throw new Error(`Expected env var SUPABASE_PRIVATE_KEY`);
+
+const url = process.env.SUPABASE_URL;
+if (!url) throw new Error(`Expected env var SUPABASE_URL`);
+
 export const run = async () => {
   const client = createClient(
     process.env.SUPABASE_URL || "",
```

Comment:
@olirice and just update these to use two vars above, sorry should have said earlier, this should become
`const client = createClient(url, privateKey);`